### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by PrivacyIsFreedom** [undergroundwires/privacy.sexy#188] [General]
Thank you very much for creating all the code, you are a modern-day hero, seriously can't thank you enough, this is a must have for any windows computer!

**Comment by Digerron** [comment]
thaks

**Comment by undergroundwires** [comment]
Thank you @PrivacyIsFreedom. Sorry that I missed this :) I watch your YouTube channel and really happy to get thumbs up ❤️. Feel free to tell your ideas or where the development should go here and I'll listen. Thank you for the content you create in your channel, you help many people.
